mcs/class/Xunit.NetCore.Extensions/Makefile

```diff
@@ -0,0 +1,17 @@
+thisdir = class/Xunit.NetCore.Extensions
```

Comment:
I'm not 100% sure why we need this. Is it because you used the xunit fork that corefx is using?

Shouldn't we be able to use stock xunit and just compile the BenchmarkAttribute etc additions into the test class?